requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Use like this:
```

Comment:
Can you please add a comment that this is for pip.

Reply:
Done